searx/engines/seekr.py

```diff
@@ -0,0 +1,104 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# lint: pylint
+"""Seekr (images, videos, news)
+"""
+
+from datetime import datetime
+from json import loads
+from urllib.parse import urlencode
+
+about = {
+    "website": 'https://seekr.com/',
+    "official_api_documentation": None,
+    "use_official_api": True,
+    "require_api_key": True,
+    "results": 'JSON',
+}
+paging = True  # news search doesn't support paging
+
+base_url = "https://api.seekr.com"
+# v2/newssearch, v1/imagetab, v1/videotab
+seekr_path = "newssearch"
+seekr_api_version = "v2"
+api_key = "srh1-22fb-sekr"
```

Comment:
I think we still have too little experience with the API to offer official settings for it today.  If we release such settings today, then we also have to maintain them in the future and in the meangtime some maintainers may have made their own settings, which would then overwrite our bugfixes again.

If it is OK for you I would like to hard-code these values (as part of my review).

----

I am not sure yet, but maybe we can get the api-key via a fetch method too

Reply:
Sure, that makes sense!
However, I don't think we can fetch an API key somehow - it's still the same one as when the seekr issue got opened here 1.5 years ago, so I think it's fine to use that one.